pkg/mover/intermediate.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+package mover
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/tarball"
+	"github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/internal"
+	"helm.sh/helm/v3/pkg/chart"
+)
+
+var (
+	// ErrNotIntermediateBundle when a verified path does not have expected intermediate bundle contents
+	ErrNotIntermediateBundle = errors.New("not an intermediate chart bundle")
+)
+
+func saveIntermediateBundle(cd *ChartData, tarFile string, log Logger) error {
+	tfw, err := newTarFileWriter(tarFile)
```

Comment:
You don't seem to be writing an temporary tarball as mentioned here https://github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/pull/96#issuecomment-982614763

Reply:
nope, because rebasing was enough work to have this up and ready for review... fixing now

Reply:
fixed now